Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -179,6 +185,7 @@ public struct Configuration: Equatable {
             }
             self.init(dict: dict)!
             configurationPath = fullPath
+            hash = yamlContents.hash
```

Comment:
Forget the bit about conforming to `Hashable`, but do you think you could adapt this to use the has value from `dict` at least? Would allow writing comments and reordering YAML contents without throwing away the cache.